silkworm/silkrpc/core/evm_executor.cpp

```diff
@@ -215,7 +215,11 @@ boost::asio::awaitable<ExecutionResult> EVMExecutor::call(
         [this, this_executor, &block, &txn, &tracers, &refund, &gas_bailout](auto&& self) {
             SILKRPC_TRACE << "EVMExecutor::call post block: " << block.header.number << " txn: " << &txn << "\n";
             boost::asio::post(workers_, [this, this_executor, &block, &txn, &tracers, &refund, &gas_bailout, self = std::move(self)]() mutable {
+                static constexpr size_t kCacheSize{5'000};
+                BaselineAnalysisCache analysis_cache{kCacheSize};
+
                 EVM evm{block, state_, config_};
+                evm.baseline_analysis_cache = &analysis_cache;
```

Comment:
Subject to measurements, but perhaps sharing a single analysis cache between coroutines can give better performance. You'll need to make `BaselineAnalysisCache` thread-safe for that though.

Also, state pool should also probably help with the performance. And `ObjectPool` already has two flavours: thread-safe and not, depending on how you use it.

Reply:
Thanks.
I will update the Sw